src/components/NewsLetter.jsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react'
+
+const NewsLetter = () => {
+  return (
+    <div className='m-14 p-4 md:flex rounded-md shadow-md justify-around bg-primary'>
```

Comment:
Excellent Work, Guys! But if you can increase the spacing from top/bottom, it would be better and nicer. It will make a difference when you include it on the homepage.